book/html.md

```diff
@@ -793,13 +794,12 @@ guess what the *right* thing is.
 
 [html5-parsing]: https://html.spec.whatwg.org/multipage/parsing.html
 
-And now for the payoff! Here is a screenshot of our very own website,
-loaded in our own browser:^[To be fair, it actually looks about the same with
-the Chapter 3 browser, but significantly better than with
-[the Chapter 2 browser](examples/example2-browserengineering-screenshot.png).]
+And now for the payoff! Figure 3 shows a screenshot of our very own [website](http://browser.engineering/),
+loaded in our own browser.^[To be fair, it actually looks about the same with the Chapter 3 browser.]
```

Comment:
Here the copyeditor wanted a screenshot of the Chapter 2 browser, to replace the link, but I decided dropping the link instead makes more sense.